### ISSUE TITLE
[QUESTION] Vapor 3

### ISSUE THREAD

**Issue by carmelogallo** [vapor/mongo-provider#26]
Hi guys! Thanks for this first of all.

I was wondering if I can use this provider with Vapor 3 like I'm already doing in Vapor 2.

Any clue?

**Comment by DanielsCode** [comment]
Any updates for vapor 3?

**Comment by vzsg** [comment]
Nope. You can use MongoKitten 4.1 directly (it's really simple with BSONEncoder/BSONDecoder, having Fluent as middleman doesn't add _that_ much), or wait for version 5, which is a complete rewrite for the post-NIO world.
